### DISCUSSION TITLE
Firebase Phone Authentication with Nuxt 3

### DISCUSSION THREAD

**Discussion by halwesit** [vuejs/vuefire#1350] [Questions and Help] [answered]
Hello Everyone, 

I was not able to find a way to implement phone authentication, does VueFire support it? if not can you suggest a way to do it?

Thanks

**Comment by posva** [comment]
VueFire won’t change anything here. Everything you need should be available in Firebase docs. You need to import the Firebase ask methods you need.

**Comment by halwesit** [comment]
Thanks for the replay, but the documentation is not very useful, can you please provide me a simple sample of phone authentication?

**Reply by posva** [reply, marked as answer]
I don't have it, since it should be in Firebase docs: https://firebase.google.com/docs/auth/web/phone-auth

But I don't have control over that.
